transmogrifier/sources/json/aardvark.py

```diff
@@ -225,17 +227,14 @@ def _range_dates(
     ) -> list[timdex.Date]:
         """Get values for issued dates."""
         range_dates = []
-        for date_range_string in [
-            date_range_strings
-            for date_range_strings in source_record.get("gbl_dateRange_drsim", [])
-        ]:
+        for date_range_string in source_record.get("gbl_dateRange_drsim", []):
             date_range_values = cls.parse_solr_date_range_string(
                 date_range_string, source_record_id
             )
             range_dates.append(
                 timdex.Date(
                     kind="Coverage",
-                    range=timdex.Date_Range(
+                    range=timdex.DateRange(
```

Comment:
I imagine this change may have been a pain to track down, but I think it's a good one!